docs/source/arguments.rst

```diff
@@ -304,3 +303,69 @@ dglke_partition
   - ``-k NUM_PARTS`` or ``--num-parts NUM_PARTS``
     The number of partitions.
 
+dglke_score
```

Comment:
i think we need to explain what this command does.

Reply:
I think we need another PR to refine our doc before 0.1.1 release

Reply:
i think it's better to call it `dglke_predict`. the main function of this command is to predict the likely head/tail entities or relations given other elements.